.github/workflows/test-suite.yaml

```diff
@@ -1,4 +1,4 @@
-name: Test Suite
+name: Build Containers and Run Tests
```

Comment:
I think this should be the same and we should change the status update places.

Reply:
`name: Build Containers and Run Tests`

Is this right?

Reply:
Ah, I understand now. I don't have permission to edit the branch protection rule.